packages/core/src/Site/index.js

```diff
@@ -1144,39 +1144,61 @@ class Site {
     options.message = this.siteConfig.deploy.message || defaultDeployConfig.message;
     options.repo = this.siteConfig.deploy.repo || defaultDeployConfig.repo;
 
-    if (travisTokenVar) {
-      if (!process.env.TRAVIS) {
-        throw new Error('-t/--travis should only be run in Travis CI.');
-      }
-      // eslint-disable-next-line no-param-reassign
-      travisTokenVar = _.isBoolean(travisTokenVar) ? 'GITHUB_TOKEN' : travisTokenVar;
-      if (!process.env[travisTokenVar]) {
-        throw new Error(`The environment variable ${travisTokenVar} does not exist.`);
+    if (ciTokenVar) {
+      const ciToken = _.isBoolean(ciTokenVar) ? 'GITHUB_TOKEN' : ciTokenVar;
```

Comment:
> I noticed that there is an extra option for tokenVar in markbind deploy --travis [tokenVar]. I have not noticed any usages of the tokenVar in the codebase. I would suggest removing tokenVar as an option if it is not being used.

the github token name may be modified https://markbind-master.netlify.app/userguide/clicommands#deploy-command (correct me if this isn't what you're referring to)

Reply:
Ahh I see. I was confused as the userGuide uses `githubTokenName` while markbind-cli uses `tokenVar`. Would it be better to change it to `markbind deploy --ci [githubTokenName]` to be consistent with the userGuide?

Reply:
Sure 👍